CMakeLists.txt

```diff
@@ -33,6 +34,14 @@ if(FORCE_COLORED_OUTPUT)
   endif()
 endif()
 
+include(GNUInstallDirs)
```

Comment:
How does Windows behave here?

Reply:
Noop on Windows. This include for future expansion and re-using already defined CMAKE_INSTALL_* variables.